src/test/features/environment-spec.ts

```diff
@@ -163,6 +163,20 @@ suite('ModelViewerElementBase with EnvironmentMixin', () => {
         expect(backgroundHasColor(scene, 'ffffff')).to.be.equal(true);
       });
 
+  test('only generates an environment when in the render tree', async () => {
+    let environmentChangeCount = 0;
+    const environmentChangeHandler = () => environmentChangeCount++;
+    element.addEventListener('environment-change', environmentChangeHandler);
+    element.style.display = 'none';
+    document.body.appendChild(element);
+    await rafPasses();
+    expect(environmentChangeCount).to.be.equal(0);
+    element.style.display = 'block';
+    await waitForEvent(element, 'environment-change');
+    expect(environmentChangeCount).to.be.equal(1);
+    element.addEventListener('environment-change', environmentChangeHandler);
```

Comment:
What does this extra addEventListener do?

Reply:
Sorry, typo. Fixed.